src/TED/TED.Utils/Tokenizer.cs

```diff
@@ -19,6 +20,8 @@ internal class Tokenizer
             { "@machineName", () => Environment.MachineName },
             { "@osVersion", () => Environment.OSVersion.ToString() },
             { "@osName", () => new ComputerInfo().OSFullName }
+            { "@dellServiceTag", () => GetSerial() }
```

Comment:
Maybe a more generic name than dellServiceTag specifically?

The Win32_BIOS query would return different results on a non-dell device.

Reply:
updated to `@machineSerial`